### PR TITLE
Duplicate keys in template

### PR DESCRIPTION
**Pull request by gustawdaniel**
fixed https://github.com/spence-s/human-object-diff/issues/26

### CODE REVIEW

**Comment by gustawdaniel** [comment]
On 30 April 2023, extended support for Node 14 LTS will end

**Comment by spence-s** [comment]
Please just do a string.replace - replaceAll unneeded here. Don't want to bring in more deps.

**Comment by spence-s** [comment]
Use the original solution you had, but instead of string.replaceAll()

do string.replace(new RegExp(token, 'g'), replacer) - which is the same as replace all. 

No looping needed for this.

File: src/sentence/index.ts [hunk 1]
```diff
@@ -53,7 +56,7 @@ export default class DiffSentence {
     let sentence = this.template
     const tokens: Token[] = ['FIELD', 'DOTPATH', 'NEWVALUE', 'OLDVALUE', 'INDEX', 'POSITION']
     for (const token of tokens) {
-      sentence = sentence.replaceAll(token, this[token])
```
Comment (spence-s) [inline src/sentence/index.ts]: remove the while loop, 

`sentence = sentence.replaceAll(new RegExp(token, 'g'), this[token])`

Which is the same as `sentence.replaceAll`

**Pull request merged**